Traceback (most recent call last):
  File "gmm_run_concrete_waveform.py", line 106, in <module>
    tmp_flow,tmp_loss=trainGMMRealNVP(
  File "/arc/project/st-tdjc-1/madmix/examples/gmm/waveform/../../../src/concrete.py", line 704, in trainGMMRealNVP
    loss = -flow.log_prob(sample).mean()
  File "/arc/project/st-tdjc-1/madmix/examples/gmm/waveform/../../../src/concrete.py", line 406, in log_prob
    return self.prior.log_prob(z) + logp
  File "/arc/project/st-tdjc-1/madmix/examples/gmm/waveform/../../../src/concrete.py", line 558, in log_prob
    xc_lp += self.invwis.log_prob(torch.moveaxis(Sigmas[k,...],2,0))
  File "/home/giank/.local/lib/python3.8/site-packages/torch/distributions/wishart.py", line 271, in log_prob
    self._validate_sample(value)
  File "/home/giank/.local/lib/python3.8/site-packages/torch/distributions/distribution.py", line 300, in _validate_sample
    raise ValueError(
ValueError: Expected value argument (Tensor of shape (1000, 2, 2)) to be within the support (PositiveDefinite()) of the distribution Wishart(covariance_matrix: torch.Size([2, 2]), df: 100.0), but found invalid values:
tensor([[[4.2160e+05, 1.0227e+02],
         [1.0227e+02, 1.8099e-01]],

        [[3.6527e+04, 8.0460e+01],
         [8.0460e+01, 6.8873e+00]],

        [[1.6126e+06, 4.5503e+01],
         [4.5503e+01, 1.2946e-02]],

        ...,

        [[2.9462e+01, 3.1714e+00],
         [3.1714e+00, 4.6699e+00]],

        [[2.7946e+03, 1.5766e+01],
         [1.5766e+01, 2.3337e-01]],

        [[3.2399e+00, 1.1506e+00],
         [1.1506e+00, 2.1919e+00]]], grad_fn=<PermuteBackward0>)
